tools/autodeployment/env.example

```diff
@@ -0,0 +1,8 @@
+# Port on which the server will be listening
+DEPLOY_PORT=
+
+# Secret to be shared between GitHub and the server
+SECRET=
+
+# Repo that will be automatically deployed
```

Comment:
Can you give an example of the format you want for this?  What is a "repo name"? (with .git? without? as a URL?)